src/pages/RFOX/hooks/useGetUnstakingRequestQuery.ts

```diff
@@ -99,6 +101,7 @@ export const useGetUnstakingRequestQuery = <SelectData = UnstakingRequest>({
   const unstakingRequestQuery = useQuery({
     queryKey,
     queryFn: getUnstakingRequestQueryFn,
+    refetchOnMount: true,
```

Comment:
Does the same as imperative `refetch`, but safe with `skipToken`.

Reply:
And it's sexier than adding another effect ourself